packages/server-core/src/user/user/user.class.ts

```diff
@@ -65,8 +65,8 @@ export class User<T = UserDataType> extends Service<T> {
       return super.find(params)
     } else if (action === 'layer-users') {
       delete params.query.action
-      params.query.instanceId = params.query.instanceId || loggedInUser.instanceId || 'intentionalBadId'
-      return super.find(params)
+      params.query.instanceId = params.query.instanceId
```

Comment:
why are we no longer falling back to `loggedInUser.instanceId` ?

Reply:
In some cases when user refresh the page  `loggedInUser.instanceId` is giving me a wrong instanceId.